src/petals/client/inference_session.py

```diff
@@ -237,7 +251,10 @@ def _enter_server_sessions(self, chosen_spans: List[RemoteSpanInfo]) -> List[_Se
         try:
             for span in chosen_spans:
                 span_uids = CHAIN_DELIMITER.join(self._sequence_manager.block_uids[span.start : span.end])
-                metadata = self._sequence_manager.get_request_metadata("rpc_inference", span_uids, peer_id=span.peer_id)
+                structure = dict()
+                metadata = self._sequence_manager.get_request_metadata(
+                    "rpc_inference", span_uids, structure, peer_id=span.peer_id
+                )
```

Comment:
Is it correct that `structure` does not get populated? Is this something we need to do in the future or is passing an empty dict here intentional?